db/migrate/20230420195221_add_assessment_delete_permissions_to_hmis_roles.rb

```diff
@@ -0,0 +1,5 @@
+class AddAssessmentDeletePermissionsToHmisRoles < ActiveRecord::Migration[6.1]
+  def change
+    add_column :hmis_roles, :can_delete_assessments, :boolean, null: false, default: false
```

Comment:
This should be done using `Hmis::Role.ensure_permissions_exist` see `db/migrate/20221028165550_add_hmis_admin_permissions.rb` for an example

Reply:
Fixed